lua/nvim-lightbulb/init.lua

```diff
@@ -393,7 +393,7 @@ NvimLightbulb.debug = function(config)
   end
 
   -- Send the LSP request
-  local context = { diagnostics = vim.lsp.diagnostic.get_line_diagnostics() }
+  local context = { diagnostics = vim.diagnostic.get() }
```

Comment:
```suggestion
  local context = { diagnostics = vim.diagnostic.get(0, { lnum = vim.fn.getcurpos()[2] }) }
```

Reply:
I add this change to my branch.  It does work the use cases I have used.